src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -240,6 +234,10 @@ class ProjectTeams extends AsyncView {
     return (
       <div>
         <SettingsPageHeader title={t('Teams')} />
+        <PanelHeader hasButtons={true}>
```

Comment:
i think technically `PanelHeader` is supposed to be inside of panel, so will fix that